packages/build/tests/plugins/fixtures/feature_flags_untrusted/plugin.js

```diff
@@ -0,0 +1,3 @@
+export const onBuild = function ({ featureFlags }) {
+  console.log("typeof featureflags:", typeof featureFlags)
```

Comment:
NIT: why not simply do a `JSON.stringify(featureFlags)` as that's what you assert?